cluster/cluster.go

```diff
@@ -260,7 +260,7 @@ type Cluster interface {
 	// It also causes this node to join the cluster.
 	// nodeInitialized indicates if the caller of this method expects the node
 	// to have been in an already-initialized state.
-	Start(clusterSize int, nodeInitialized bool) error
+	Start(clusterSize int, nodeInitialized bool, gossipPort string) error
```

Comment:
not sure why is port being passed as a string ?

Reply:
it's used as a string. why convert from int to string if we don't have to?

Reply:
it needs to be validated if its a string.